internal/resources/resource_host.go

```diff
@@ -220,6 +221,12 @@ func hostSchema() map[string]*schema.Schema {
 			Optional:    true,
 			Description: "map of label name to label value for this host",
 		},
+		hISCSIDiscoveryIP: {
+			Type:        schema.TypeString,
```

Comment:
It is a list, so use "Type:     schema.TypeList,", and you need to define the "Elem". Look at other TypeList above.

Reply:
No longer a list. I have shown VMaaS and CaaS team that one Discovery will work.